spec/views/flags/new.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe "flags/new", type: :view do
+  before(:each) do
+    assign(:flag, Flag.new(
+      :name => "MyString",
```

Comment:
Indent the first parameter one step more than `Flag.new(`.<br>Use the new Ruby 1.9 hash syntax.